watson_developer_cloud/iam_token_manager.py

```diff
@@ -0,0 +1,142 @@
+import requests
+import time
+
+DEFAULT_IAM_URL = 'https://iam.bluemix.net/identity/token'
+CONTENT_TYPE = 'application/x-www-form-urlencoded'
+ACCEPT = 'application/json'
+DEFAULT_AUTHORIZATION = 'Basic Yng6Yng='
+REQUEST_TOKEN_GRANT_TYPE = 'urn:ibm:params:oauth:grant-type:apikey'
+REQUEST_TOKEN_RESPONSE_TYPE = 'cloud_iam'
+REFRESH_TOKEN_GRANT_TYPE = 'refresh_token'
+
+class IAMTokenManager(object):
+    def __init__(self, iam_api_key=None, iam_access_token=None, iam_url=None):
+        self.iam_api_key = iam_api_key
+        self.user_access_token = iam_access_token
+        self.url = iam_url if iam_url else DEFAULT_IAM_URL
```

Comment:
iam_url or if you don't want to use `iam` remove it from `iam_api_key` too.

Reply:
Changing the class variable to `iam_url`